libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -435,6 +452,8 @@ public void addProgressChangeListener(@NonNull MapboxNavigation navigation) {
   }
 
   /**
+   * @deprecated
+   *
```

Comment:
You can safely remove this https://kotlinlang.org/docs/reference/kotlin-doc.html#block-tags

> KDoc does not support the @deprecated tag. Instead, please use the @Deprecated annotation.

```suggestion
```

Reply:
CI complains if it's not there:   [ant:checkstyle] [ERROR] /root/code/libnavigation-ui/src/main/java/com/******/navigation/ui/map/NavigationMapboxMap.java:464: Must include both @java.lang.Deprecated annotation and @deprecated Javadoc tag with description. [MissingDeprecated]

Reply:
Ohh, it's Java. Thought `NavigationMapboxMap` was already in Kotlin.

Could we add a comment to the `@deprecated` tag?

Reply:
Added